### PR TITLE
Fix Umbreon ex, remove ef debug from Alakazam MT

### PR DESCRIPTION
**Pull request by starg09**
Breaks the engine and doesn't permit the other testing to be done. Returns null anyways so rip this.

### CODE REVIEW

File: src/tcgwars/logic/impl/gen3/UnseenForces.groovy [hunk 1]
```diff
@@ -795,18 +795,27 @@ public enum UnseenForces implements LogicCardInfo {
...
+                  wcu "Intimidating Ring prevents attack"
+                  prevent()
+                }
               }
             }
-          }
-          getterA (IS_ABILITY_BLOCKED) { Holder h->
-            if (self.active && h.effect.target.owner != self.owner && !h.effect.target.evolution && h.effect.ability instanceof PokePower) {
-              h.object=true
+            eff2 = getter (IS_ABILITY_BLOCKED) { Holder h->
```
Comment (flagrama) [inline src/tcgwars/logic/impl/gen3/UnseenForces.groovy:808]: You changed this from a `getterA` to `getter`. Not sure that was intentional.

Comment (flagrama) [reply src/tcgwars/logic/impl/gen3/UnseenForces.groovy:808]: Actually, think you did this with many pokepowers. Is this the way it's supposed to be done on abilities?

Comment (starg09) [reply src/tcgwars/logic/impl/gen3/UnseenForces.groovy:808]: Yeah, had to register it manually so I can re-check abilities on deactivation of the Poké-Body.

Comment (starg09) [reply src/tcgwars/logic/impl/gen3/UnseenForces.groovy:808]: > Actually, think you did this with many pokepowers. Is this the way it's supposed to be done on abilities?

Luckily, modern abilities don't get deactivated (most of the time) but entirely removed from the game while the ability is on. However, on that topic GALARIAN_WEEZING from RCL is an example of getters being used this way.

File: src/tcgwars/logic/impl/gen3/UnseenForces.groovy [hunk 2]
```diff
@@ -2499,12 +2508,20 @@ public enum UnseenForces implements LogicCardInfo {
...
+                if (h.effect.ability instanceof PokePower || h.effect.ability instanceof PokeBody) {
+                  h.object=true
+                }
               }
             }
+            new CheckAbilities().run(bg)
+          }
+          onDeactivate {
+            eff.unregister()
+            new CheckAbilities().run(bg)
```
Comment (axpendix) [inline src/tcgwars/logic/impl/gen3/UnseenForces.groovy:2524]: actually, `getterA`,  `onActivate` and `onDeactivate` can all coexist in a single ability definition. @starg09

Comment (starg09) [reply src/tcgwars/logic/impl/gen3/UnseenForces.groovy:2524]: Oh? I stand corrected then. Will revert this later, sorry!

Comment (axpendix) [reply src/tcgwars/logic/impl/gen3/UnseenForces.groovy:2524]: The order of run is: 
- actionA registered [only a single one allowed]
- delayedA runs [only a single one allowed]
- getterA's run [multiples allowed]
- onActivate called

**Pull request merged**